### PR TITLE
Good

### PR DESCRIPTION
**Pull request by Desynkova**
ms-vscode.go",    "ms-vscode.vscode-typescript-tslint-plugin",    "esbenp.prettier-vscode"  ]

### CODE REVIEW

**Review by Desynkova — COMMENTED** [review]
Good

File: js/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "walletlink",
-  "version": "2.1.0",
```
Comment (chrissyjai) [inline js/package.json:3]:
```suggestion
  "version": "2.1.0",
```

File: js/src/WalletLink.ts [hunk 2]
```diff
@@ -2,14 +2,14 @@
 // Copyright (c) 2018-2020 Coinbase, Inc. <https://www.coinbase.com/>
 // Licensed under the Apache License, version 2.0
 
-import { WalletLinkProvider } from "./provider/WalletLinkProvider"
```
Comment (chrissyjai) [inline js/src/WalletLink.ts:5]: Bitcoin

**Review by chrissyjai — APPROVED** [review]

File: js/src/WalletLink.ts (hunk 2 above)
Comment (ShortyV21) [reply js/src/WalletLink.ts:5]: Yeah I'm lost I don't know how to do all that

**Pull request closed without merging**